incubator/basic-auth/basic-auth/values.yaml

```
@@ -0,0 +1,53 @@
+# SLATE instance tag
+Instance: ''
+
+# SSSD Configuration
+# This is an entire file which maps directly to /etc/sssd/sssd.conf
+# TODO (More Info)
+sssd: |+
+  [sssd]
+  config_file_version = 2
+  reconnection_retries = 3
+  sbus_timeout = 30
+  services = nss, pam
+  domains = ad.utah.edu
+
+  [nss]
+  filter_groups = root
+  filter_users = root
+  reconnection_retries = 3
+
+  [pam]
+  reconnection_retries = 3
+
+  [domain/ad.utah.edu]
+  debug_level = 3
+  enumerate = false
+  cache_credentials = true
+  use_fully_qualified_names = false
+
+  auth_provider = krb5
+  chpass_provider = krb5
+  krb5_realm = AD.UTAH.EDU
+  krb5_server = kdc1.chpc.utah.edu:88,kdc2.chpc.utah.edu:88,kdc3.chpc.utah.edu:88,kdc4.chpc.utah.edu:88,kdc5.chpc.utah.edu:88
+  krb5_auth_timeout = 15
+  krb5_store_password_if_offline = true
+  krb5_ccname_template = FILE:%d/krb5cc_%U_XXXXXX
+  krb5_ccachedir = /tmp
+
+  id_provider = ldap
+  access_provider = ldap
+  ldap_uri = ldap://ldap.chpc.utah.edu,ldap://ldap2.chpc.utah.edu
+  ldap_search_base = dc=chpc,dc=utah,dc=edu
+  ldap_tls_cacert = /etc/pki/tls/certs/ca-bundle.crt
+  ldap_tls_reqcert = never
+  ldap_krb5_init_creds = true
+  ldap_krb5_ticket_lifetime = 86400
+  ldap_schema = rfc2307bis
+  ldap_group_member = uniqueMember
+
+  override_homedir = /home/%u
+  default_shell = /bin/bash
+  
+  # Restrict to user's who's primary group is CHPC.
+  ldap_access_filter = (gidNumber=1722)
```

```suggestion
  ldap_access_filter = (uidNumber=*)
```
Mentioned this in person but this filter ensures only those working at the CHPC can be auth'd through SSSD. This should probably be changed to the above.